packages/image_picker/image_picker_platform_interface/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 2.5.0
+
+* Re-adds `requestFullMetadata` option, but as a parameter of a new `pickImageFromSource`
```

Comment:
Let's mention all the changes in this PR in details, such as:
Deprecated `getImage` (We should do this too in this PR)
Added `getImageFromSource`
Added `ImagePickerOptions`, 
Added `requestFullMetadata`